### PR TITLE
Bringing cloud sql sample up to standards, and fixing a few other sta…

### LINKED ISSUE #111: Bring appengine/cloudsql up to standards

### PR DESCRIPTION
**Pull request by jonparrott**
…ndards issues in other samples

closes #111.

/cc @waprin.

### CODE REVIEW

**Comment by jonparrott** [comment]
Build failed due to transient error, ignore travis.

File: appengine/bigquery/README.md [hunk 1]
```diff
@@ -2,18 +2,24 @@
...
-  [here](https://cloud-dot-devsite.googleplex.com/bigquery/authorization#clientsecrets).
-  Once you've downloaded the client's json secret, copy it to the root directory
-  of this project, and rename it to `client_secrets.json`.
-* You can then run the sample on your development server:
+2. You must then update `main.py` and replace `<your-project-id>` with your project's
+   ID.
 
-    $ dev_appserver.py .
+3. You'll need a client id from your project - instructions
+   [here](https://cloud.google.com/bigquery/authorization#clientsecrets).
```
Comment (jerjou) [inline appengine/bigquery/README.md]: Since you're here, can you change '/authorization' to '/authentication'? (...it's a long story..)

**Comment by jerjou** [comment]
LGTM

File: tox.ini [hunk 2]
```diff
@@ -18,6 +18,7 @@ commonargs =
 [testenv:gae]
 deps =
     {[testenv]deps}
+    mysql-python==1.2.5
```
Comment (waprin) [inline tox.ini]: wondering why it can't just go in requirements-dev.txt?

Comment (jonparrott) [inline tox.ini]: It's only for GAE. May move it into requirements-dev.txt at a later date.

**Pull request merged**